<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head profile="http://gmpg.org/xfn/11">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Accepted Posters | Research in Computational Molecular Biology 2016</title>
<link rel="stylesheet" href="wp-content/themes/retina/style.css" type="text/css" media="all" />
<link rel="pingback" href="xmlrpc.php" />
<link rel="alternate" type="application/rss+xml" title="Research in Computational Molecular Biology 2016 &raquo; Feed" href="feed/" />
<link rel="alternate" type="application/rss+xml" title="Research in Computational Molecular Biology 2016 &raquo; Comments Feed" href="comments/feed/" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/72x72\/","ext":".png","source":{"concatemoji":"http:\/\/recomb2016.bioinformatics.ucla.edu\/wp-includes\/js\/wp-emoji-release.min.js?ver=4.4.2"}};
			!function(a,b,c){function d(a){var c,d=b.createElement("canvas"),e=d.getContext&&d.getContext("2d"),f=String.fromCharCode;return e&&e.fillText?(e.textBaseline="top",e.font="600 32px Arial","flag"===a?(e.fillText(f(55356,56806,55356,56826),0,0),d.toDataURL().length>3e3):"diversity"===a?(e.fillText(f(55356,57221),0,0),c=e.getImageData(16,16,1,1).data.toString(),e.fillText(f(55356,57221,55356,57343),0,0),c!==e.getImageData(16,16,1,1).data.toString()):("simple"===a?e.fillText(f(55357,56835),0,0):e.fillText(f(55356,57135),0,0),0!==e.getImageData(16,16,1,1).data[0])):!1}function e(a){var c=b.createElement("script");c.src=a,c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var f,g;c.supports={simple:d("simple"),flag:d("flag"),unicode8:d("unicode8"),diversity:d("diversity")},c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.simple&&c.supports.flag&&c.supports.unicode8&&c.supports.diversity||(g=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",g,!1),a.addEventListener("load",g,!1)):(a.attachEvent("onload",g),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),f=c.source||{},f.concatemoji?e(f.concatemoji):f.wpemoji&&f.twemoji&&(e(f.twemoji),e(f.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='retina-css-960-css'  href='wp-content/themes/retina/lib/css/960.css?ver=4.4.2' type='text/css' media='all' />
<link rel='stylesheet' id='retina-google-fonts-css'  href='http://fonts.googleapis.com/css?family=Open+Sans%7CComfortaa&#038;ver=4.4.2' type='text/css' media='all' />
<link rel='stylesheet' id='jetpack_css-css'  href='wp-content/plugins/jetpack/css/jetpack.css?ver=3.9.2' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery.js?ver=1.11.3'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min.js?ver=1.2.1'></script>
<link rel='https://api.w.org/' href='wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 4.4.2" />
<link rel="canonical" href="accepted-posters.html" />
<link rel='shortlink' href='http://wp.me/P71ox3-7Q' />
<link rel="alternate" type="application/json+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2016.bioinformatics.ucla.edu%2Faccepted-posters%2F" />
<link rel="alternate" type="text/xml+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2016.bioinformatics.ucla.edu%2Faccepted-posters%2F&#038;format=xml" />
<style type='text/css'>img#wpstats{display:none}</style>	
		<style type="text/css">
    #headimg {
        width: 100%;
		height: 215px;
        background: url(wp-content/uploads/sites/7/2015/06/cropped-Screen-Shot-2015-06-29-at-12.56.22-PM2.png) no-repeat top center;
		-webkit-background-size: cover;
		-moz-background-size: cover;
		-o-background-size: cover;
		background-size: cover;
    }
    </style>
    	
		


<!-- Jetpack Open Graph Tags -->
<meta property="og:type" content="article" />
<meta property="og:title" content="Accepted Posters" />
<meta property="og:url" content="accepted-posters.html" />
<meta property="og:description" content="Accepted Posters &nbsp; Rufus Vinod, Syeda Yumna Farrukh and Roquyya Gul. Human VEGF-A121 Dimerization by SUMO Fusion Technology in Prokaryotic System Franziska Singer, Michael Prummer, Miquel Busqâ€¦" />
<meta property="article:published_time" content="2016-03-07T07:44:20+00:00" />
<meta property="article:modified_time" content="2016-04-17T18:26:23+00:00" />
<meta property="og:site_name" content="Research in Computational Molecular Biology 2016" />
<meta property="og:image" content="https://s0.wp.com/i/blank.jpg" />
<meta property="og:locale" content="en_US" />
<meta name="twitter:card" content="summary" />
</head>
<body class="page page-id-486 page-template-default">
<div class="wrapper">
  
  <div id="header">  
    
    <div id="headimg"> 
  
  
</div>
          
    <div id="nav">
      <div class="container_16 clearfix">
        <div class="grid_16 alpha omega">
            <div class="menu clearfix"><ul id="menu-page-structure" class="sf-menu"><li id="menu-item-47" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-47"><a href="index.html">Home</a></li>
<li id="menu-item-49" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-49"><a href="about-recomb.html">About RECOMB</a></li>
<li id="menu-item-295" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-295"><a href="registration.html">Registration</a>
<ul class="sub-menu">
	<li id="menu-item-299" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-299"><a href="travel-support.html">Travel Support</a></li>
	<li id="menu-item-272" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-272"><a href="conference-venue.html">Conference Venue</a></li>
	<li id="menu-item-310" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-310"><a href="hotel.html">Hotel Information</a></li>
</ul>
</li>
<li id="menu-item-108" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-108"><a href="committees.html">Committees</a></li>
<li id="menu-item-81" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-81"><a href="papers-and-posters.html">Submit to RECOMB</a>
<ul class="sub-menu">
	<li id="menu-item-141" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-141"><a href="paper-submissions.html">Paper Submission</a></li>
	<li id="menu-item-138" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-138"><a href="posters.html">Poster Submission</a></li>
	<li id="menu-item-293" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-293"><a href="highlights-submission.html">Highlights Submission</a></li>
</ul>
</li>
<li id="menu-item-399" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-399"><a href="keynote-speakers.html">Program</a>
<ul class="sub-menu">
	<li id="menu-item-436" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-436"><a href="conference-schedule.html">Conference Schedule</a></li>
	<li id="menu-item-352" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-352"><a href="accepted-papers.html">Accepted Papers</a></li>
	<li id="menu-item-488" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-486 current_page_item menu-item-488"><a href="accepted-posters.html">Accepted Posters</a></li>
</ul>
</li>
<li id="menu-item-473" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-473"><a href="keynote-speakers-2.html">Keynote Speakers</a></li>
<li id="menu-item-131" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-131"><a href="satellite-meetings.html">Satellite Meetings</a>
<ul class="sub-menu">
	<li id="menu-item-132" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-132"><a href="satellite-meetings/recomb-seq.html">RECOMB-Seq</a></li>
	<li id="menu-item-529" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-529"><a href="recomb-seq-schedule.html">RECOMB-SEQ Schedule</a></li>
	<li id="menu-item-134" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-134"><a href="satellite-meetings/recomb-ccb.html">RECOMB-CCB</a></li>
	<li id="menu-item-530" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-530"><a href="recomb-ccb-schedule.html">RECOMB-CCB Schedule</a></li>
	<li id="menu-item-133" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-133"><a href="satellite-meetings/recomb-genetics.html">RECOMB-Genetics</a></li>
	<li id="menu-item-491" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-491"><a href="recomb-genetics-schedule.html">RECOMB-Genetics Schedule</a></li>
</ul>
</li>
<li id="menu-item-430" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-430"><a href="sponsors.html">Sponsors</a></li>
<li id="menu-item-302" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-302"><a href="co-founder-symposium.html">Waterman Symposium</a></li>
<li id="menu-item-376" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-376"><a href="bioinformatics-flipped-course.html">Bioinformatics Flipped Course</a></li>
</ul></div>        </div>
      </div>
    </div>
  
  </div>
<div class="container_16 clearfix">
  
  <div class="grid_10 alpha">
    <div id="content">	  
	  
	        
                
          <div id="post-486" class="post-486 page type-page status-publish hentry">
  
    <h1 class="entry-title entry-title-single entry-title-page">Accepted Posters</h1>
  
    
  <div class="entry-content clearfix">
  	<div class="kcite-section" kcite-section-id="486">
<p><strong>Accepted Posters</strong></p>
<p>&nbsp;</p>
<ol>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Rufus Vinod, Syeda Yumna Farrukh and Roquyya Gul. <strong>Human VEGF-A121 Dimerization by SUMO Fusion Technology in Prokaryotic System</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Franziska Singer, Michael Prummer, Miquel Busquets Lopez, Christian Stirnimann and Daniel Stekhoven. <strong>NEXUS: supporting personalized medicine with state-of-the art technologies</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Franziska Kreuchwig, Sebastiaan van Heesch, Eleonora Adami, Allison Faber, Sebastian SchÃ¤fer, Matthias Heinig, Giannino Patone, Susanne Blachut, Anita MÃ¼ller, Michal Pravenec and Norbert HÃ¼bner. <strong>QTL mapping to elucidate genetic basis of translational regulation in heart failure</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Roni Wilentzik and Irit Gat-Viks. <strong>The genetic basis of individual variation in transcriptional responses: a network-based approach</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jie Ren, Fengzhu Sun, Gesine Reinert, Minghua Deng, Kai Song and Chuck Cannon. <strong>Inference of Markovian properties of molecular sequences from NGS data and applications to comparative genomics</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Wangshu Zhang, Marcelo P. Coba and Fengzhu Sun. <strong>Inference of domain-disease associations from domain-protein, protein-disease and disease-disease relationships</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Brian Peng He. <strong>Gene regulation in different cell states: from chromatin structure to transcription dynamics</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Daqing Wang, Amy Webb and Audrey Papp. <strong>Transcriptome sequencing analysis in human brain regions: protein-coding and non-coding RNAs, isoforms and alleles</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Ashley Cass, Jae Hoon Bahn, Xianzhi Lin, Jae-Hyung Lee, Christopher Greer, Yun-Hua Esther Hsiao and Xinshu Xiao. <strong>Endogenous small RNAs target coding and noncoding transcripts to induce endonucleolytic cleavage</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Marcin Åšwistak and Jerzy Tiuryn. <strong>TCRÎ± enhancer substructures are conserved in the genomes of mouse and human</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Bo Zhang and Yi-Chieh Wu. <strong>Bayesian Estimation of Evolutionary Histories under Duplication, Loss, and Coalescence</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Fabian Schmich, Gunter Merdes and Niko Beerenwinkel.<strong> netprioR: Integrative graphâ€“based prioritisation of perturbation screen hits</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Debjit Ray. <strong>Predictive Pathogen Biology: Genome-Based Prediction of Pathogenic Potential and Countermeasures Targets</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Yun-Hua Esther Hsiao, Jae Hoon Bahn, Xianzhi Lin, Tak-Ming Chan, Rena Wang and Xinshu Xiao. <strong>Alternative Splicing Modulated by Genetic Variants Demonstrates Accelerated Evolution Regulated by Highly Conserved Proteins</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Christos Dimitrakopoulos and Niko Beerenwinkel. <strong>Interaction Network Analysis links tumor mutations to expression patterns</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Hatice Osmanbeyoglu, Eneda Toska, Jose Baselga and Christina Leslie. <strong>Modeling the impact of somatic alterations in human cancers</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Ying-Tsang Lo and Tun-Wen Pai. <strong>Prediction of Conformational Epitope using a Novel Spiral Feature Vector</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Dmitrii Meleshko, Hosein Mohimani, Marnix Medema and Pavel Pevzner. <strong>npSPAdes: a new tool for biosynthetic gene cluster discovery from metagenomic data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Aliz Rao and Stanley Nelson. <strong>Calculating the statistical significance of rare variants causal for Mendelian disorders</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Julia Salzman. <strong>Statistical Detection of circular and linear RNA variants</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Nadav Rappoprt and Atul Butte. <strong>Predict demographic features from lab tests</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Huy Dinh, Yaping Liu, Fides Lay, Peter Jones and Benjamin Berman. <strong>A software toolkit for NOMe-seq</strong> <strong>including single-molecule analysis of DNA methylation and nucleosome positioning</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Sangwoo Kim and Hojung Nam. <strong>The role of loss-of-function variants in ethnic specific enzymatic genes</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Sumana Srivatsa, Fabian Schmich, Jack Kuipers and Niko Beerenwinkel. <strong>Pathway reconstruction from combinatorial gene knockdowns exploiting siRNA off-target effects</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Simon Dirmeier and Niko Beerenwinkel.<strong> Integrated identification of positive-sense RNA-viral dependency factors from perturbation data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Tommy Hansen and Fabio Vandin. <strong>NoMAS: An Efficient Algorithm to Find Mutated Subnetworks Associated with Survival in Cancer</strong></span></span></li>
<li style="font-weight: 400"><del><span style="font-weight: 400"><span style="font-weight: 400">Cong-Doan Truong, Hung-Cuong Trinh and Yung-Keun Kwon. <strong>Topological and Dynamical Characteristics of a Large-Scale Signaling Network by Module Analysis</strong></span></span>&nbsp;
<p></del></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Mithun Mitra, Elizabeth L. Johnson, David C. Corney, David G. Robinson, Daniel G. Taylor, Wei Wang, Sandra Batista and Hilary A. Coller. <strong>Cell cycle exit to a quiescent state involves alternative isoform usage</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Danny Zeevi, Joshua Bloom, Ephrat Levy-Lahad, David Zangen and Leonid Kruglyak. <strong>Analysis of the genetic basis of height in large Jewish pedigrees</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Qi Yang, Joseph Boland, Michael Cullen and Meredith Yeader. <strong>HPV_ NGS, detecting and annotating genetics variant of HPV16 whole Genome sequences</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Lisa Gai, Serghei Mangul, Eun Yong Kang, Susana Eyheramendy and Eleazar Eskin. <strong>Accounting for linkage disequilibrium when estimating the heritability contribution of a genomic region</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Minseung Kim, Navneet Rai, Violeta Zorraquino, Xiaokang Wang and Ilias Tagkopoulos. <strong>Multi-omics integration and optimal experimental design accurately predicts cellular and growth dynamics</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Artur Jaroszewicz and Jason Ernst. <strong>Probabilistic modeling of parallel ChIP-Seq data enables discrimination of direct and indirect binding events</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Reazur Rahman, Gung-Wei Chirn, Abhay Kanodia and Nelson Lau. <strong>Unique transposon landscapes are pervasive across Drosophila melanogaster genomes</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Monika Sekelja, Jonas Paulsen and Philippe Collas. <strong>Revealing cell subpopulations and their chromatin structures by applying deconvolution on Hi-C and ChIP-seq data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Ortal Shnaps, Eyal Perry, Dana Silverbush and Roded Sharan.<strong>Â Inference ofÂ PersonalizedÂ DrugÂ TargetsÂ viaÂ NetworkÂ Propagation</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Shan Sabri, Jason Ernst and Kathrin Plath. <strong>High-throughput Profiling of Single-cells Reprogramming to Pluripotency</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Dong-Yeon Cho, Hangnoh Lee, Damian Wojtowicz, Steven Russell, Brian Oliver and Teresa M. Przytycka. <strong>Interplay between copy number, dosage compensation and expression noise in Drosophila</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Federica Martina, Francesca Cordero, Emilio Siena, Gianfranco Balbo and Duccio Medini. <strong>A multilayer gene-network predicts the immune quality response to vaccination</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Douglas Arneson, Anindya Bhattacharya, Le Shu, Ville-Petteri Makinen and Xia Yang. <strong>Mergeomics: A Webserver for Identifying Pathological Pathways, Networks, and Key Regulators via Omics Data Integration</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Forough Firoozbakht, Iman Rezaeian, Alioune Ngom and Luis Rueda. <strong>Breast Cancer Network Biomarker Identification by Incorporating Genomics, Transcriptomics and Proteomics Data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Sang Cheol Kim and Seong Beom Cho. <strong>COEX-seq: COnvert a variety of measurements of gene EXpression in RNA-seq</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Erfan Sayyari and Siavash Mirarab. <strong>Fast coalescent-based computation of local branch support from quartet frequencies</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Pan-Gyu Kim, Gunhwan Ko, Jongcheol Yoon, Byungwook Lee and Ryan W. Kim. <strong>The workflow system based on Hadoop for bio-big data analysis</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Bingqiang Liu and Qin Ma. <strong>Development of Computational Tools in DNA motif identification and analyses</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Ali Yousefian-Jazi and Jinwook Choi. <strong>FCM-EM: A Novel Approach for Discovering DNA Motifs</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Thomas Conway, Benjamin Goudey and Justin Bedo.<strong> K-mer prefixes for alignment free homology on bacterial data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Yoichi Takenaka, Shigeto Seno and Hideo Matsuda. <strong>Dynamic Analysis of Gene Regulations using Leaving-One-Out Expression Profile</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Suresh B Vepuri and Mahmoud E Soliman.<strong> Development of New lead molecule for the inhibition of human Aldose Reductase- Virtual screening, X-ray structure and Induced fit Docking</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Kazumasa Saito, Shigeto Seno, Yoichi Takenaka and Hideo Matsuda. <strong>Bayesian Network Inference from gene expression profiles with a small number of samples</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Nan Hua, Long Pei, Ke Gong, Harianto Tjong, Chao Dai, Qingjiao Li, Xianghong Jasmine Zhou and Frank Alber. <strong>Unraveling human genome architecture with population-based modeling</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Robert SchÃ¶pflin, Guillaume Andrey, Verena Heinrich, Daniel Ibrahim, Christina Paliou, BjÃ¸rt Kragesteen, Myriam Hochradel, Malte Spielmann, Stefan Mundlos and Martin Vingron. <strong>Characterizing the regulatory landscapes of limb-associated genes using Capture-C</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Phelim Bradley and Dr Zamin Iqbal.<strong> Mykrobe predictor: Rapid antibiotic-resistance predictions from genome sequence data using de Bruijn graphs</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Yazhou Sun and Mikaela Rosen.<strong> Identification of Novel Functional Elements Inside Human Genome with RNA-Seq</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Menglu Li, Angel C.Y. Mak, Ernest T. Lam, Pui-Yan Kwok, Kevin Y. Yip, Ting Fung Chan and Siu Ming Yiu. <strong>Towards a more accurate error model for BioNano optical maps</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Norbert Dojer. <strong>Learning Bayesian networks from datasets joining continuous and discrete variables</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Junho Kim, Ju Heon Maeng, Jae Seok Lim, Hyeonju Son, Junehawk Lee, Jeong Ho Lee and Sangwoo Kim. <strong>Systematic identification of false somatic variants caused by vector contamination</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Pasquale Laise, Pietro Lo Riso, Vivek Das, Michela Lupia, Fabrizio Bianchi, Nicoletta Colombo, Ugo Cavallaro and Giuseppe Testa. <strong>Reconstruction of the epigenetic regulatory circuits in ovarian cancer</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jochen Singer, Katharina Jahn, Jack Kuipers and Niko Beerenwinkel. <strong>Allele frequency tree reconstruction for single-cell data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Rafal Zaborowski and Bartek Wilczynski. <strong>New method for systematic detection of Hi-C differential interactions</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Michael Cheng, Sheng-Yuan Chiu, Menglu Li, Siu Ming Yiu, Liqing Zhang and Wing-Kai Hon. <strong>WebRTG: A Web Front-end for RNA Structure Prediction using Rectangle Tree Gramma</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Vinaya Vijayan, Siu Ming Yiu and Liqing Zhang. <strong>Framework for integration of genome and exome data to improve the identification of somatic variants</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Johannes Helmuth, Li Na and Ho-Ryun Chung. <strong>normR: Regime enrichment calling for ChIP-seq data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Murodzhon Akhmedov, Roberto Montemanni and Ivo Kwee.<strong> Application of Steiner Trees and Deep Learning for DLBCL Cancer subtype Classification</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jongin Lee, Mikang Sim and Jaebum Kim. <strong>Efficient pairwise whole-genome alignment by utilizing existing alignments of relevant species</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Anton Bankevich, Dmitry Antipov, Sergey Nurk and Pavel Pevzner. <strong>New members of SPAdes assembly tools family</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Geethika Yalamanchili and Christopher Rao.<strong> Selective promoter recognition by RpoS: A Biological Puzzle</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Serghei Mangul, Nicolas Strauli, Harry Yang, Ryan Hernandez, Roel Ophoff, Eleazar Eskin and Noah Zaitlen. <strong>Examining lost reads to survey the microbiome and immune components of the human body across 43 human sites from 175 individuals</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Charles Labuzzetta, Margaret Antonio, Dennis Watson and Paul Anderson. <strong>MVPq: A novel filtering method for isoform-based features from alternative splicing events enhances phenotype prediction accuracy via machine learning techniques in RNA-Seq experiments</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Yana Safonova, Alexander Shlemov, Andrey Bzikadze, Sergey Bankevich and Pavel Pevzner. <strong>A novel toolkit for analysis of adaptive immune repertoires using immunosequencing and mass spectra data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Idoia Ochoa, Mikel Hernaez, Rachel Goldfeder, Tsachy Weissman and Euan Ashley. <strong>Denoising of Quality Scores for Boosted Inference and Reduced Storage</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Adriana Sperlea and Jason Ernst. <strong>Systematic prediction of putative regulatory regions from multiple sequence alignments</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Chee Lee, Maureen Sartor, Laura Scott and Yanxiao Zhang.<strong> Transposons, segmental duplications and sequence mappability, and their relationship with gene length and function</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Margaret Antonio, Charles Labuzzetta, Dennis Watson and Paul Anderson. <strong>METRO: A pipeline for comparative meta-analysis and visualization of gene and pathway expression in RNA-Seq experiments</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Rachelly Normand, Renaud Gaujoux, Elina Starosvetsky, Gali Shalev-Malul and Shai Shen-Orr. <strong>Found In Translation: Statistical model for improved translation from mouse gene expression to human biology</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Kedar Tatwawadi, Mikel Hernaez, Idoia Ochoa and Tsachy Weissman.<strong> Fast retrieval from compressed collections of genomic variants</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Dominik Budday, Sigrid Leyendecker and Henry van Den Bedem. <strong>Clash- and constraint-guided motion planning reveals conformational transition pathways in proteins</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Petko Fiziev and Jason Ernst. <strong>Systematic Detection of Spatiotemporal Patterns of Epigenetic Changes</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Daniel Hogan, Bharathikumar Vellalore, Clarence Geyer and Anthony Kusalik.<strong> An algorithm for hierarchical clustering of short sequences generated from antibody phage display experiments</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Alexander Goncearenco, Stefan Wuchty, Peter Uetz and Anna R. Panchenko. <strong>Protein-protein interactomes with structurally resolved interfaces</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Huo Li, Steven Lockton, Soma Ray, Gilda Shayan, Norm Allaire, Mark Cosentino, Shirley Philips, Jaya Goyal, Chris Roberts, Tim Harris, Martin Beaulieu and John Carulli. <strong>Identification of a microRNA biomarker signature that differentiates relapsing-remitting Multiple Sclerosis patients from healthy volunteers</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Olaitan Awe, Segun Fatumo and Angela Makolo. <strong>Genetic Interrelationships between Zika Virus, Dengue Virus, Chikungunya Virus and Yellow Fever Virus Strains</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Alexander Goncearenco, Stephanie L. Rager, Quing-Xiang Sang, Igor B. Rogozin and Anna R. Panchenko. <strong>Probabilistic context-specific models of somatic mutations in cancer</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Julia Herman-Izycka, Michal Wlasnowolski and Bartek Wilczynski. <strong>Predicting mammalian enhancers from sequence can be biased by the information specific to promoter sequences</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Weiling Li, Raunaq Malhotra, Mary Poss and Raj Acharya. <strong>Mining mapped reads in 1000 Genomes data for accurately predicting polymorphic human endogenous retroviruses</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">David Seifert and Niko Beerenwinkel. <strong>Aligning NGS reads of viral populations experiencing high insertion-deletion rates</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Mario Banuelos, Rubi Almanza, Lasith Adhikari, Suzanne Sindi and Roummel Marcia. <strong>Predicting Structural Variants In Genomic Data Using Sparsity and Relatedness Constraints</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Steven Lakin, Rob Raymond, Noelle Noyes, Enrique Doster, Zaid Abdo, Keith Belk, Paul Morley and Christina Boucher. <strong>Quantification of antimicrobial resistance profiles from metagenomic sequencing data using Hidden Markov Models and post hoc filtering</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Agnieszka Mykowiecka, Pawel Szczesny and Pawel Gorecki. <strong>Inference of gene-species relationships using tree reconciliation</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Aleksandra Gruca and Marek Sikora. <strong>RuleGO2 &#8211; the algorithm and platform for verification of user hypothesis of co-existence of functional pathways in gene set description</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Ilia Minkin, Son Pham and Paul Medvedev. <strong>TwoPaCo: An efficient algorithm to build the compacted de Bruijn graph from many complete genomes</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jimin Song, Chicheng Zhang, Kamalika Chaudhuri and Kevin Chen. <strong>Spectral Learning Algorithms for Comparative Epigenomics</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Renaud Gaujoux, Francesco Vallania, Naama Meimon, Elina Starosvetsky, Edmond Sago, Yehuda Chowers, Purvesh Khatri and Shai Shen-Orr.<strong> Immune cell subset signatures make for robust predictive biomarkers in IBD</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Rares Turliuc, Bogdan Grecu and Adriana Climescu-Haulica. <strong>Dynamic classification based on human Gene Damage Index: a physiological approach</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Raunaq Malhotra, Manjari Mukhopadhyay, Mary Poss and Raj Acharya. <strong>A frame-based representation of genomic sequences for removing errors and detecting low frequency variants in samples</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Yang Lu, Ting Chen, Jed Fuhrman and Fengzhu Sun. <strong>COCACOLA: binning metagenomic contigs using sequence COmposition, read CoverAge, CO-alignment, and paired-end read LinkAge</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Katelyn Hughes, Oleksandr Narykov, Nathan Johnson and Dmitry Korkin. <strong>FUELCAN: Determining the oncogenic potential of a fusion event via semi-supervised learning</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Yacidzohara Rodriguez, Ricardo GonzÃ¡lez-Mendez and Carmen Cadilla. <strong>Evolution of the Twist Subfamily Vertebrate Proteins: Discovery of a Signature Motif and Origin of the Twist1 Glycine-rich Motifs in the Amino-terminus Disordered Domain</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Sajad Mirzaei and Yufeng Wu. <strong>A New Method for Inferring Local Genealogical Tree Topologies from Haplotypes with the Presence of Recombination</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Abed Alkhateeb, Iman Rezaeian and Luis Rueda.<strong> Nonnegative Matrix Factorization Clustering for Time Series Cancer Progression Data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jennifer Zou, Eun Yong Kang, Serghei Mangul, Eyal Ben-David, Buhm Han, Aldons Lusis, Sagiv Shifman, Jason Ernst and Eleazar Eskin. <strong>Utilizing allele specific expression to identify cis-regulatory variants</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Xinan Yang, Bin Wang and John Cunningham. <strong>Effective integration of genome-wide histone modification with transcriptional profiles</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jennifer Rogers, Andrew Fishberg, Nora Youngs and Yi-Chieh Wu.<strong> Reconciliation Feasibility in the Presence of Gene Duplications, Losses, and Coalescence with Multiple Individuals per Species</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Noelle Noyes, Maggie Weinroth, Enrique Doster, Pablo Rovira Sanz, Xiang Yang, Jaime Ruiz, Christina Boucher, Kenneth Jones, Paul Morley, Keith Belk and Zaid Abdo. <strong>A Dirichlet-Multinomial Bayesian Model for Evaluating Antimicrobial Resistance Gene Abundance in Metagenomic Data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Harry Yang, Serghei Mangul, Farhad Hormozdiari, Alexandra Dainis, Elizabeth Tseng, Euan Ashley, Alex Zelikovsky and Eleazar Eskin. <strong>Hercules: Haplotype-aware error correction of long single molecule reads in RNA-Seq</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Han Li and Peter Ralph. <strong>Using Local PCA to Investigate Population Structure Along the Genome</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Bahar Behsaz, Hosein Mohimani, Pieter C. Dorrestein and Pavel Pevzner. <strong>Automated Peptidic Natural Product Sequencing Using Mass Spectrometry and Marfey&#8217;s Analysis</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Abdullah Khamis, Artem V. Artemov, Anna Lioznova, Vladimir B. Bajic and Yulia Medvedeva.<strong> Regulatory role of single CpG methylation</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Rasmus Fonseca, Sune G. Enevoldsen and Martin A. Hansen. <strong>SeqScan: Scan nucleotide and protein sequences with elaborate user-defined patterns</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Olivera Grujic, Yuju Lee and Jason Ernst.<strong> Systematic Prediction of Conserved Non-Exonic Bases from Epigenomic and Transcription Factor Binding Data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jasmijn A. Baaijens, Amal Zine El Aabidine, Eric Rivals and Alexander Schoenhuth. <strong>De novo assembly of viral quasispecies</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Davide Verzotto, Audrey Teo, Luka Sterbic, Burton Chia, Mile Sikic, Axel Hillmer and Niranjan Nagarajan. <strong>Super-scaffolding of large eukaryotic genomes with single molecule maps</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Guillaume Holley and Faraz Hach.<strong> BFT-Comp: Alignment-free and reference-free compression of pan-genome sequencing reads</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Mikel Hernaez, Pan Li, Idoia Ochoa, Tsachy Weissman and Olgica Milencovic.<strong> Micro: Microbial Database Compression Via Community Detection and Reference Discovery</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Noelle Noyes, Maggie Weinroth, Enrique Doster, Pablo Rovira Sanz, Chris Dean, Xiang Yang, Jaime Ruiz, Kenneth Jones, Christina Boucher, Zaid Abdo, Keith Belk and Paul Morley.<strong> A HaplotypeÂ IdentificationÂ System forÂ AntimicrobialÂ ResistanceÂ Genes inÂ ShotgunÂ MetagenomicÂ Data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Chao Wang, Xi Li, Changlong Li, Aili Wang and Xuehai Zhou. <strong>MIC++: Accelerating Maximal Information Coefficient Calculation with Clouds, GPUs, and FPGAs</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Shu-Hwa Chen, Sheng-Yao Su, Yi-Hsuan Lu and Chung-Yen Lin. <strong>MOLAS: Multi-Omics onLine Analysis System for Gene Expression Profiling</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jeremie Kim, Damla Senol, Donghyuk Lee, Mohammed Alser, Hasan Hassan, Oguz Ergin, Can Alkan and Onur Mutlu.<strong> Genome Read In-Memory Mapper (GRIMM): Fast Location Filtering with 3D Stacked Memory Technologies</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Mohammadhossein Moeinzadeh, Jun Yang and Martin Vingron. <strong>Haplotype Reconstruction for Sweet Potato (Ipomoea Batatas [L.] Lam) genome</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Mohammad Sadeh.<strong> Estimating Causal Effect Strength between Histone Modifications and Modifiers in Chromatin Signaling Networks</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Min Xu and Frank Alber. <strong>De novo visual proteomics in single cells through pattern mining</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Sanjana Gupta, Raghavendran Partha and Jocelyn Sunseri. <strong>Assessment of Classification Algorithm Performance on Cancer-Derived Gene Expression Data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Derek Aguiar, Li-Fang Cheng, Fantine Mordelet, Bianca Dumitrascu and Barbara Engelhardt. <strong>iDISQ: Isoform discovery and individual-specific quantification from RNA-seq data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Aziz Mezlini, Tal Friedman, Ekansh Sharma and Anna Goldenberg.<strong> Improving power for exome variant aggregation using biologically informed graphical models</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Florian Massip, Misha Sheinman, Sophie Schbath and Peter Arndt. <strong>Comparing the Statistical Fate of Paralogous and Orthologous Sequences</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Ascia Eskin, Richard Wang, Hane Lee, Valerie Arbodela, Kevin Squire, Joshua L. Deignan, Negar Khanlou, Perry B. Shieh and Stanley F. Nelson.<strong> RNA sequencing improves diagnostic yield in Duchenne Muscular Dystrophy</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Chelsea J.-T. Ju, Ruirui Li, Zhengliang Wu, Zhao Yang and Wei Wang. <strong>Fleximer: Accurate Quantification of RNA-Seq via Variable Length k-mers</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Qian Yang, Yue Hu, Jun Li and Xuegong Zhang. <strong>UFsQTL: an Ultra-Fast Method of Splicing QTL</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Gleb Kichaev, Huwenbo Shi, Megan Roytman and Bogdan Pasaniuc. <strong>Large-scale integrative fine-mapping over 20 complex traits with over 2.2 million phenotypic measurements</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Huwenbo Shi, Gleb Kichaev and Bogdan Pasaniuc. <strong>Fine-scale mapping of genomic heritability using summary association statistics</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Megan Roytman and Bogdan Pasaniuc.<strong> Methods for fine-mapping with chromatin and expression data</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Jeongkyun Kim and Hyunju Lee. <strong>Text-mining approaches for extracting disease-gene-biological events relationships for all diseases</strong></span></span></li>
<li style="font-weight: 400"><span style="font-weight: 400"><span style="font-weight: 400">Mikhail Kolmogorov, Brian Raney, Duncan Odom, Paul Flicek, David Thybert, Benedict Paten and Son Pham. <strong>Pseudo-chromosome assembly of large and complex genomes using multiple references</strong></span></span></li>
<li style="font-weight: 400">Alexander Shlemov, Andrey Bzikadze, Sergey Bankevich and Yana Safonova. <strong>IgQUAST: A quality assessment tool for adaptive immune repertoires</strong></li>
<li style="font-weight: 400">Sergey Bankevich, Yana Safonova, Alexander Shlemov and Andrey Bzikadze.<strong> IgReC: constructing adaptive immune repertoire from barcoded sequencing reads</strong></li>
<li style="font-weight: 400">Gryte Satas, Mohammed El-Kebir and Ben Raphael. <strong><strong>Estimating the Cancer Cell Fraction of Mutations from Bulk Tumor Sequencing Data</strong></strong></li>
<li style="font-weight: 400">Rebecca Elyanow, Hsin-Ta Wu and Benjamin J. Raphael. <strong><strong>Identifying Structural Variants from 10X Synthetic Long Read Sequencing Data</strong></strong></li>
<li style="font-weight: 400">Andrey Bzikadze, Yana Safonova, Alexander Shlemov and Sergey Bankevich. <strong><strong>AntEvolo: a novel approach for joint recombination and clonal analysis of antibody repertoires</strong></strong></li>
<li style="font-weight: 400">Hiren Karathia and Sridhar Hannenhalli. <strong>Global transcriptional dysregulation in Breast cancer</strong></li>
<li style="font-weight: 400">Azim Dehghani Amirabad and Marcel H. Schulz. <strong>Multitask regression for context-specific prioritization of miRNA targets in transcripts</strong></li>
<li style="font-weight: 400">Vincent Kyi, Brian Bleakley and Howard Choi. <strong>BD2K Centers of Excellence</strong></li>
<li style="font-weight: 400">Vasilisa Rudneva, Simon Anders, Wolfgang Huber and Jan Korbel.Â <strong>Robust identification of recurrent intergenic somatic mutations in cancer genomes by cross-validation</strong></li>
</ol>
<!-- kcite active, but no citations found -->
</div> <!-- kcite-section 486 -->  </div> <!-- end .entry-content -->
  
    
</div> <!-- end #post-486 .post_class -->

<div id="comments">
  
  
  
  
</div><!-- #comments -->        
              
            
    </div> <!-- end #content -->
  </div> <!-- end .grid_10 -->
  
  <div class="grid_6 omega">
  <div id="sidebar">
  
	<div id="text-4" class="widget widget_text widget-widget_text clearfix"><div class="widget-wrap widget-inside"><h3 class="widget-title">RECOMB Schedule At-a-Glance</h3>			<div class="textwidget"><p>April 16th &amp; 17th, 2016- UCLA Campus<br />
Satellite Workshops:<br />
RECOMB-CCB<br />
RECOMB-Seq<br />
RECOMB-Genetics</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
1:30 p.m. to 6 p.m.<br />
Mike Waterman Symposium</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
2:00pm â€“ 5:00pm<br />
Bioinformatics Flipped Course</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
6:00 p.m. to 9 p.m.<br />
RECOMB 2016 Welcome Reception</p>
<p>April 18th-21st, 2016- Loews, Santa Monica<br />
RECOMB 2016 Conference</p>
<p>Special thanks to our sponsors...</p>
<p>- International Society for Computational Biology<br />
- Polish BioinformaticsÂ Society<br />
- 10x Genomics<br />
- USC Dornsife College of Letters, Arts and Sciences<br />
- UCLA Clinical Translational Science Institute (CTSI)<br />
- UCLA Institute for Quantitative and Computational Biosciences<br />
- Intel<br />
- PLOS Computational Biology<br />
- UCLA Department of Computer Science</p>
</div>
		</div></div>  
  </div> <!-- end #sidebar -->
</div>  <!-- end .grid_6 -->
</div> <!-- end .container_16 -->
  
  <div id="footer">
    <div class="container_16">
      <div class="grid_11">
  &copy; Copyright 2018 - <a href="index.html">Research in Computational Molecular Biology 2016</a></div>
<div class="grid_5 grid_credit">
  Retina Theme by <a href="http://wpaisle.com/" title="WPAisle">WPAisle</a> &sdot; <a href="http://wordpress.org/" title="WordPress">WordPress</a>
</div>
    </div>
  </div>

</div> <!-- end .wrapper -->
<!-- Kcite Plugin Installed: Disabled as there are no shortcodes-->
	<div style="display:none">
	</div>
<script type='text/javascript' src='http://s0.wp.com/wp-content/js/devicepx-jetpack.js?ver=201808'></script>
<script type='text/javascript' src='wp-content/themes/retina/lib/js/superfish/superfish-combine.min.js?ver=1.5.9'></script>
<script type='text/javascript' src='wp-content/themes/retina/lib/js/common.js?ver=1.0'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min.js?ver=4.4.2'></script>
<script type='text/javascript' src='http://stats.wp.com/e-201808.js' async defer></script>
<script type='text/javascript'>
	_stq = window._stq || [];
	_stq.push([ 'view', {v:'ext',j:'1:3.9.2',blog:'103766985',post:'486',tz:'0',srv:'recomb2016.bioinformatics.ucla.edu'} ]);
	_stq.push([ 'clickTrackerInit', '103766985', '486' ]);
</script>
</body>
</html>