<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head profile="http://gmpg.org/xfn/11">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Accepted Papers | Research in Computational Molecular Biology 2016</title>
<link rel="stylesheet" href="wp-content/themes/retina/style.css" type="text/css" media="all" />
<link rel="pingback" href="xmlrpc.php" />
<link rel="alternate" type="application/rss+xml" title="Research in Computational Molecular Biology 2016 &raquo; Feed" href="feed/" />
<link rel="alternate" type="application/rss+xml" title="Research in Computational Molecular Biology 2016 &raquo; Comments Feed" href="comments/feed/" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/72x72\/","ext":".png","source":{"concatemoji":"http:\/\/recomb2016.bioinformatics.ucla.edu\/wp-includes\/js\/wp-emoji-release.min.js?ver=4.4.2"}};
			!function(a,b,c){function d(a){var c,d=b.createElement("canvas"),e=d.getContext&&d.getContext("2d"),f=String.fromCharCode;return e&&e.fillText?(e.textBaseline="top",e.font="600 32px Arial","flag"===a?(e.fillText(f(55356,56806,55356,56826),0,0),d.toDataURL().length>3e3):"diversity"===a?(e.fillText(f(55356,57221),0,0),c=e.getImageData(16,16,1,1).data.toString(),e.fillText(f(55356,57221,55356,57343),0,0),c!==e.getImageData(16,16,1,1).data.toString()):("simple"===a?e.fillText(f(55357,56835),0,0):e.fillText(f(55356,57135),0,0),0!==e.getImageData(16,16,1,1).data[0])):!1}function e(a){var c=b.createElement("script");c.src=a,c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var f,g;c.supports={simple:d("simple"),flag:d("flag"),unicode8:d("unicode8"),diversity:d("diversity")},c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.simple&&c.supports.flag&&c.supports.unicode8&&c.supports.diversity||(g=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",g,!1),a.addEventListener("load",g,!1)):(a.attachEvent("onload",g),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),f=c.source||{},f.concatemoji?e(f.concatemoji):f.wpemoji&&f.twemoji&&(e(f.twemoji),e(f.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='retina-css-960-css'  href='wp-content/themes/retina/lib/css/960.css?ver=4.4.2' type='text/css' media='all' />
<link rel='stylesheet' id='retina-google-fonts-css'  href='http://fonts.googleapis.com/css?family=Open+Sans%7CComfortaa&#038;ver=4.4.2' type='text/css' media='all' />
<link rel='stylesheet' id='jetpack_css-css'  href='wp-content/plugins/jetpack/css/jetpack.css?ver=3.9.2' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery.js?ver=1.11.3'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min.js?ver=1.2.1'></script>
<link rel='https://api.w.org/' href='wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 4.4.2" />
<link rel="canonical" href="accepted-papers.html" />
<link rel='shortlink' href='http://wp.me/P71ox3-5E' />
<link rel="alternate" type="application/json+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2016.bioinformatics.ucla.edu%2Faccepted-papers%2F" />
<link rel="alternate" type="text/xml+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2016.bioinformatics.ucla.edu%2Faccepted-papers%2F&#038;format=xml" />
<style type='text/css'>img#wpstats{display:none}</style>	
		<style type="text/css">
    #headimg {
        width: 100%;
		height: 215px;
        background: url(wp-content/uploads/sites/7/2015/06/cropped-Screen-Shot-2015-06-29-at-12.56.22-PM2.png) no-repeat top center;
		-webkit-background-size: cover;
		-moz-background-size: cover;
		-o-background-size: cover;
		background-size: cover;
    }
    </style>
    	
		


<!-- Jetpack Open Graph Tags -->
<meta property="og:type" content="article" />
<meta property="og:title" content="Accepted Papers" />
<meta property="og:url" content="accepted-papers.html" />
<meta property="og:description" content="All of the RECOMB Proceedings papers are available at  &nbsp; Hormozdiari, Farhad, Fereydoun Hormozdiari, Carl Kingsford, Paul Medvedev, and Fabio Vandin. The Second Decade of the International Con…" />
<meta property="article:published_time" content="2015-12-24T00:33:29+00:00" />
<meta property="article:modified_time" content="2016-05-12T18:10:10+00:00" />
<meta property="og:site_name" content="Research in Computational Molecular Biology 2016" />
<meta property="og:image" content="https://s0.wp.com/i/blank.jpg" />
<meta property="og:locale" content="en_US" />
<meta name="twitter:card" content="summary" />
</head>
<body class="page page-id-350 page-template-default">
<div class="wrapper">
  
  <div id="header">  
    
    <div id="headimg"> 
  
  
</div>
          
    <div id="nav">
      <div class="container_16 clearfix">
        <div class="grid_16 alpha omega">
            <div class="menu clearfix"><ul id="menu-page-structure" class="sf-menu"><li id="menu-item-47" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-47"><a href="index.html">Home</a></li>
<li id="menu-item-49" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-49"><a href="about-recomb.html">About RECOMB</a></li>
<li id="menu-item-295" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-295"><a href="registration.html">Registration</a>
<ul class="sub-menu">
	<li id="menu-item-299" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-299"><a href="travel-support.html">Travel Support</a></li>
	<li id="menu-item-272" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-272"><a href="conference-venue.html">Conference Venue</a></li>
	<li id="menu-item-310" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-310"><a href="hotel.html">Hotel Information</a></li>
</ul>
</li>
<li id="menu-item-108" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-108"><a href="committees.html">Committees</a></li>
<li id="menu-item-81" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-81"><a href="papers-and-posters.html">Submit to RECOMB</a>
<ul class="sub-menu">
	<li id="menu-item-141" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-141"><a href="paper-submissions.html">Paper Submission</a></li>
	<li id="menu-item-138" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-138"><a href="posters.html">Poster Submission</a></li>
	<li id="menu-item-293" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-293"><a href="highlights-submission.html">Highlights Submission</a></li>
</ul>
</li>
<li id="menu-item-399" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-399"><a href="keynote-speakers.html">Program</a>
<ul class="sub-menu">
	<li id="menu-item-436" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-436"><a href="conference-schedule.html">Conference Schedule</a></li>
	<li id="menu-item-352" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-350 current_page_item menu-item-352"><a href="accepted-papers.html">Accepted Papers</a></li>
	<li id="menu-item-488" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-488"><a href="accepted-posters.html">Accepted Posters</a></li>
</ul>
</li>
<li id="menu-item-473" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-473"><a href="keynote-speakers-2.html">Keynote Speakers</a></li>
<li id="menu-item-131" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-131"><a href="satellite-meetings.html">Satellite Meetings</a>
<ul class="sub-menu">
	<li id="menu-item-132" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-132"><a href="satellite-meetings/recomb-seq.html">RECOMB-Seq</a></li>
	<li id="menu-item-529" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-529"><a href="recomb-seq-schedule.html">RECOMB-SEQ Schedule</a></li>
	<li id="menu-item-134" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-134"><a href="satellite-meetings/recomb-ccb.html">RECOMB-CCB</a></li>
	<li id="menu-item-530" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-530"><a href="recomb-ccb-schedule.html">RECOMB-CCB Schedule</a></li>
	<li id="menu-item-133" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-133"><a href="satellite-meetings/recomb-genetics.html">RECOMB-Genetics</a></li>
	<li id="menu-item-491" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-491"><a href="recomb-genetics-schedule.html">RECOMB-Genetics Schedule</a></li>
</ul>
</li>
<li id="menu-item-430" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-430"><a href="sponsors.html">Sponsors</a></li>
<li id="menu-item-302" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-302"><a href="co-founder-symposium.html">Waterman Symposium</a></li>
<li id="menu-item-376" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-376"><a href="bioinformatics-flipped-course.html">Bioinformatics Flipped Course</a></li>
</ul></div>        </div>
      </div>
    </div>
  
  </div>
<div class="container_16 clearfix">
  
  <div class="grid_10 alpha">
    <div id="content">	  
	  
	        
                
          <div id="post-350" class="post-350 page type-page status-publish hentry">
  
    <h1 class="entry-title entry-title-single entry-title-page">Accepted Papers</h1>
  
    
  <div class="entry-content clearfix">
  	<div class="kcite-section" kcite-section-id="350">
<p>All of the RECOMB Proceedings papers are available at <a href="http://link.springer.com/book/10.1007/978-3-319-31957-5." target="_blank">http://link.springer.com/book/10.1007/978-3-319-31957-5.</a></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">Hormozdiari, Farhad, Fereydoun Hormozdiari, Carl Kingsford, Paul Medvedev, and Fabio Vandin.<br />
</span><i><span style="font-weight: 400">The Second Decade of the International Conference on Research in Computational Molecular Biology</span></i><span style="font-weight: 400">            </span><span style="font-weight: 400"><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_1">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_1</a><br />
Data for the 20th anniversary article is available here:<br />
<a href="https://docs.google.com/spreadsheets/d/19_X2RWwXfJXkrvTCAHCgZHSKxB_OgZ_dTs7T_MVotBM/edi" target="_blank">https://docs.google.com/spreadsheets/d/19_X2RWwXfJXkrvTCAHCgZHSKxB_OgZ_dTs7T_MVotBM/edi</a>t</span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">Antipov, Dmitry, Nolan Hartwick, Max Shen, Michael Rayko, Alla Lapidus and Pavel Pevzner.<br />
</span><i><span style="font-weight: 400">plasmidSPAdes: Assembling Plasmids from Whole Genome Sequencing Data<br />
</span></i><span style="font-weight: 400">           <a href="http://biorxiv.org/content/early/2016/04/15/048942">http://biorxiv.org/content/early/2016/04/15/048942</a></span></p>
<p><span style="font-weight: 400">Artyomenko, Alexander, Nicholas Wu, Serghei Mangul, Eleazar Eskin, Ren Sun and Alex Zelikovsky.<br />
</span><i><span style="font-weight: 400">Long single-molecule reads can resolve the complexity of the Influenza virus composed of rare, closely related mutant variants           </span></i><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_12"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_12</span></a></p>
<p><span style="font-weight: 400">Calviello, Lorenzo, Neelanjan Mukherjee, Emanuel Wyler, Henrik Zauber, Antje Hirsekorn, Matthias Selbach, Markus Landthaler, Benedikt Obermayer and Uwe Ohler.<br />
</span><i><span style="font-weight: 400">Detecting actively translated open reading frames in ribosome profiling data</span></i><span style="font-weight: 400">            </span><a href="http://dx.doi.org/10.1038/nmeth.3688"><span style="font-weight: 400">http://dx.doi.org/10.1038/nmeth.3688</span></a></p>
<p><span style="font-weight: 400">Cristea, Simona, Jack Kuipers and Niko Beerenwinkel.<br />
</span><i><span style="font-weight: 400">pathTiMEx: Joint Inference of Mutually Exclusive Cancer Pathways and their Dependencies in Tumor Progression            </span></i><a href="http://link.springer.com/chapter/10.1007%2F978-3-319-31957-5_5"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007%2F978-3-319-31957-5_5</span></a></p>
<p><span style="font-weight: 400">Dao, Phuong, Jan Hoinka, Yijie Wang, Mayumi Takahashi, Jiehua Zhou, Fabrizio Costa, John Rossi, John Burnett, Rolf Backofen and Teresa Przytycka.<br />
</span><i><span style="font-weight: 400">AptaTRACE: Elucidating Sequence-Structure Binding Motifs by Uncovering Selection Trends in HT-SELEX Experiments<br />
</span></i><span style="font-weight: 400"><i>           </i></span><a href="http://biorxiv.org/content/early/2016/04/05/047357"><span style="font-weight: 400">http://biorxiv.org/content/early/2016/04/05/047357</span></a></p>
<p><span style="font-weight: 400">Donmez, Nilgun, Salem Malikic, Alexander Wyatt, Colin Collins, Martin Gleave and Cenk Sahinalp.<br />
</span><i><span style="font-weight: 400">Clonality inference from single tumor samples using low coverage sequence data</span></i><span style="font-weight: 400">            </span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_6"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_6</span></a></p>
<p><span style="font-weight: 400">El-Kebir, Mohammed, Gryte Satas, Layla Oesper and Ben Raphael.<br />
</span><i><span style="font-weight: 400">Multi-State Perfect Phylogeny Mixture Deconvolution and Applications to Cancer Sequencing</span></i><span style="font-weight: 400">            </span> <i><span style="font-weight: 400">           </span></i><a href="http://arxiv.org/abs/1604.02605"><span style="font-weight: 400">http://arxiv.org/abs/1604.02605</span></a></p>
<p><span style="font-weight: 400">Ezpeleta, Joaquín, Flavia J. Krsticevic, Pilar Bulacio and Elizabeth Tapia.<br />
</span><i><span style="font-weight: 400">Designing robust watermark barcodes for multiplex long-read sequencing</span></i><span style="font-weight: 400"><br />
<i>           </i></span><a href="http://arxiv.org/abs/1604.01344"><span style="font-weight: 400">http://arxiv.org/abs/1604.01344</span></a></p>
<p><span style="font-weight: 400">Fusi, Nicolo and Jennifer Listgarten.<br />
</span><i><span style="font-weight: 400">Flexible Modelling of Genetic Effects on Function-Valued Traits</span></i> <a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_7"><span style="font-weight: 400"><br />
</span></a><span style="font-weight: 400"><i>           </i></span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_7"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_7</span></a></p>
<p><span style="font-weight: 400">Gupta, Nitish, Komal Sanjeev, Tim Wall, Carl Kingsford and Rob Patro.<br />
</span><i><span style="font-weight: 400">Efficient Index Maintenance Under Dynamic Genome Modification</span></i><span style="font-weight: 400"><br />
<i>           </i></span><a href="http://arxiv.org/abs/1604.03132"><span style="font-weight: 400">http://arxiv.org/abs/1604.03132</span></a></p>
<p><span style="font-weight: 400">Hallen, Mark, Jonathan Jou and Bruce Donald.<br />
</span><i><span style="font-weight: 400">LUTE (Local Unpruned Tuple Expansion): Accurate continuously flexible protein design with general energy functions and rigid-rotamer-like efficiency<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_9">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_9</a></p>
<p><span style="font-weight: 400">Hansen, Tommy and Fabio Vandin.<br />
</span><i><span style="font-weight: 400">Finding Mutated Subnetworks Associated with Survival Time in Cancer</span></i> <a href="http://arxiv.org/abs/1604.02467"><span style="font-weight: 400"><br />
</span></a><span style="font-weight: 400"><i>           </i></span><a href="http://arxiv.org/abs/1604.02467"><span style="font-weight: 400">http://arxiv.org/abs/1604.02467</span></a></p>
<p><span style="font-weight: 400">Hashemifar, Somaye, Qixing Huang and Jinbo Xu.<br />
</span><i><span style="font-weight: 400">Joint alignment of multiple protein-protein interaction networks via convex optimization</span></i><span style="font-weight: 400">            </span><a href="http://arxiv.org/abs/1604.03482"><span style="font-weight: 400">http://arxiv.org/abs/1604.03482</span></a></p>
<p><span style="font-weight: 400">Jahn, Katharina, Jack Kuipers and Niko Beerenwinkel.<br />
</span><i><span style="font-weight: 400">Tree inference for single-cell data<br />
</span></i><a href="http://biorxiv.org/content/early/2016/04/09/047795"><span style="font-weight: 400">http://biorxiv.org/content/early/2016/04/09/047795</span></a></p>
<p><span style="font-weight: 400">Kolmogorov, Mikhail, Eamonn Kennedy, Zhuxin Dong, Gregory Timp and Pavel Pezvner.<br />
</span><i><span style="font-weight: 400">Single-Molecule Protein Identification by Sub-Nanopore Sensors<br />
</span></i><span style="font-weight: 400">          </span><a style="line-height: 1.5" href="http://arxiv.org/abs/1604.02270">http://arxiv.org/abs/1604.02270</a></p>
<p><span style="font-weight: 400">Kowada, Luis Antonio B., Daniel Doerr, Simone Dantas and Jens Stoye.<br />
</span><i><span style="font-weight: 400">New Genome Similarity Measures based on Conserved Gene Adjacencies</span></i><span style="font-weight: 400">            </span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_15"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_15</span></a></p>
<p><span style="font-weight: 400">Kshirsagar, Meghana, Jaime Carbonell, Judith Klein-Seetharaman and Keerthiram Murugesan.<br />
</span><i><span style="font-weight: 400">Multitask matrix completion for learning protein interactions across diseases</span></i><span style="font-weight: 400">            </span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_4"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_4</span></a></p>
<p><span style="font-weight: 400">Kwon, Sunyoung, Gyuwan Kim, Byunghan Lee, Sungroh Yoon and Young-Han Kim.<br />
</span><i><span style="font-weight: 400">NASCUP: Nucleic Acid Sequence Classification by Universal Probability</span></i><span style="font-weight: 400">  </span><a href="http://arxiv.org/abs/1511.04944"><span style="font-weight: 400"><br />
</span></a><span style="font-weight: 400"><i>           </i></span><a href="http://arxiv.org/abs/1511.04944"><span style="font-weight: 400">http://arxiv.org/abs/1511.04944</span></a></p>
<p><span style="font-weight: 400">Li, Yang, Shiguo Zhou, David Schwartz and Jian Ma.<br />
</span><i><span style="font-weight: 400">Allele-Specific Quantification of Structural Variations in Cancer Genomes</span></i><span style="font-weight: 400"><br />
<i>           </i></span><a href="http://dx.doi.org/10.1101/048207"><span style="font-weight: 400">http://dx.doi.org/10.1101/048207</span></a></p>
<p><span style="font-weight: 400">Lin, Yu, Max W. Shen, Jeffrey Yuan and Pavel Pevzner.<br />
</span><i><span style="font-weight: 400">Assembly of Long Error-Prone Reads Using de Bruijn Graphs<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://dx.doi.org/10.1101/048413">http://dx.doi.org/10.1101/048413</a></p>
<p><span style="font-weight: 400">Lu, Yang, Ting Chen, Jed Fuhrman and Fengzhu Sun.<br />
</span><i><span style="font-weight: 400">COCACOLA: binning metagenomic contigs using sequence COmposition, read CoverAge, CO-alignment, and paired-end read LinkAge<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://arxiv.org/abs/1604.02512">http://arxiv.org/abs/1604.02512</a></p>
<p><span style="font-weight: 400">Luo, Yunan, Jianyang Zeng, Bonnie Berger and Jian Peng.<br />
</span><i><span style="font-weight: 400">Low-density locality-sensitive hashing boosts metagenomic binning</span></i><span style="font-weight: 400"><br />
</span><span style="font-weight: 400"><i>           </i></span><a href="http://arxiv.org/abs/1604.02699"><span style="font-weight: 400">http://arxiv.org/abs/1604.02699</span></a></p>
<p><span style="font-weight: 400">Ma, Xiuli, Guangyu Zhou, Jingjing Wang, Jian Peng and Jiawei Han.<br />
</span><i><span style="font-weight: 400">Complexes Detection in Biological Networks via Diversified Dense Subgraphs Mining</span></i><span style="font-weight: 400"><br />
<i>           </i></span><a href="http://publish.illinois.edu/guangyuzhou/files/2015/11/recomb.pdf"><span style="font-weight: 400">http://publish.illinois.edu/guangyuzhou/files/2015/11/recomb.pdf</span></a></p>
<p><span style="font-weight: 400">Maryanto Gunawan, Andreas Dwi, Bhaskar Dasgupta and Louxin Zhang.<br />
</span><i><span style="font-weight: 400">Locating a Tree in a Reticulation-Visible Network in Cubic Time<br />
</span></i><a href="http://arxiv.org/abs/1507.02119"><span style="font-weight: 400">http://arxiv.org/abs/1507.02119</span></a></p>
<p><span style="font-weight: 400">Nurk, Sergey, Dmitry Meleshko and Pavel Pevzner.<br />
</span><i><span style="font-weight: 400">metaSPAdes: a new versatile de novo metagenomics assembler<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://arxiv.org/abs/1604.03071">http://arxiv.org/abs/1604.03071</a></p>
<p><span style="font-weight: 400">Pellow, David, Darya Filippova and Carl Kingsford.<br />
</span><i><span style="font-weight: 400">Improving Bloom filter performance on sequence data using k-mer Bloom filters</span></i><span style="font-weight: 400">            </span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_10"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_10</span></a></p>
<p><span style="font-weight: 400">Popic, Victoria and Serafim Batzoglou.<br />
</span><i><span style="font-weight: 400">Efficient Privacy-Preserving Read Mapping Using Locality Sensitive Hashing and Secure Kmer Voting</span></i><span style="font-weight: 400">            </span><a href="http://dx.doi.org/10.1101/046920"><span style="font-weight: 400">http://dx.doi.org/10.1101/046920</span></a></p>
<p><span style="font-weight: 400">Rajagopal, Nisha, Sharanya Srinivasan, Kameron Kooshesh, Yuchun Guo, Matthew Edwards, Budhaditya Banerjee, Tahin Syed, Bart Emons, David Gifford and Richard Sherwood.<br />
</span><i><span style="font-weight: 400">High-throughput mapping of regulatory DNA<br />
</span></i><a href="http://www.nature.com/nbt/journal/v34/n2/abs/nbt.3468.html"><span style="font-weight: 400">http://www.nature.com/nbt/journal/v34/n2/abs/nbt.3468.html</span></a></p>
<p><span style="font-weight: 400">Rozov, Roye, Aya Brown Kav, David Bogumil, Itzhak Mizrahi, Eran Halperin and Ron Shamir.<br />
</span><i><span style="font-weight: 400">Recycler: an algorithm for detecting plasmids from de novo assembly graphs</span></i><span style="font-weight: 400">            </span><a href="http://biorxiv.org/content/early/2016/04/05/029926"><span style="font-weight: 400">http://biorxiv.org/content/early/2016/04/05/029926</span></a></p>
<p><span style="font-weight: 400">Sahlin, Kristoffer, Mattias Frånberg and Lars Arvestad.<br />
</span><i><span style="font-weight: 400">Structural variation detection with read pair information — An improved null-hypothesis reduces bias</span></i><span style="font-weight: 400">            </span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_13"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_13</span></a></p>
<p><span style="font-weight: 400">Salmela, Leena, Riku Walve, Eric Rivals and Esko Ukkonen.<br />
</span><i><span style="font-weight: 400">Accurate selfcorrection of errors in long reads using de Bruijn graphs</span></i><span style="font-weight: 400"><br />
<i>           </i></span><a href="http://arxiv.org/abs/1604.02233"><span style="font-weight: 400">http://arxiv.org/abs/1604.02233</span></a></p>
<p><span style="font-weight: 400">Sefer, Emre and Ziv Bar-Joseph.<br />
</span><i><span style="font-weight: 400">Shall we dense? Comparing design strategies for time series expression experiments</span></i><span style="font-weight: 400">            </span><a href="http://www.cs.cmu.edu/~esefer/papers/gene_recomb.pdf"><span style="font-weight: 400">http://www.cs.cmu.edu/~esefer/papers/gene_recomb.pdf</span></a></p>
<p><span style="font-weight: 400">Shao, Mingfu and Bernard Moret.<br />
</span><i><span style="font-weight: 400">On Computing Pairwise Breakpoint Distances<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_14">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_14</a></p>
<p><span style="font-weight: 400">Shlemov, Alexander, Sergey Bankevich, Andrey Bzikadze and Yana Safonova.<br />
</span><i><span style="font-weight: 400">IgReC 2.0: New algorithmic challenges of adaptive immune repertoire construction</span></i><span style="font-weight: 400">            </span><a href="http://arxiv.org/pdf/1604.02193.pdf"><span style="font-weight: 400">http://arxiv.org/pdf/1604.02193.pdf</span></a></p>
<p><span style="font-weight: 400">Simmons, Sean, Cenk Sahinalp and Bonnie Berger.<br />
</span><i><span style="font-weight: 400">Enabling Privacy-Preserving GWAS in Heterogenous Human Populations</span></i><span style="font-weight: 400">            </span><a href="http://groups.csail.mit.edu/cb/PrivGWAS/PrivStrat.pdf"><span style="font-weight: 400">http://groups.csail.mit.edu/cb/PrivGWAS/PrivStrat.pdf</span></a></p>
<p><span style="font-weight: 400">Singh, Shashank, Sabrina Rashid, Saket Navlakha and Ziv Bar-Joseph.<br />
</span><i><span style="font-weight: 400">Distributed Gradient Descent in Bacterial Food Search<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://arxiv.org/abs/1604.03052">http://arxiv.org/abs/1604.03052</a></p>
<p><span style="font-weight: 400">Sobih, Ahmed, Alexandru I. Tomescu and Veli Mäkinen.<br />
</span><i><span style="font-weight: 400">MetaFlow: Metagenomic profiling based on whole-genome coverage analysis with min-cost flows</span></i><span style="font-weight: 400">            </span> <i><span style="font-weight: 400">           </span></i><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_8"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_8</span></a></p>
<p><span style="font-weight: 400">Srivastava, Avi, Hirak Sarkar, Laraib Malik, and Robert Patro.<br />
</span><i><span style="font-weight: 400">Accurate, Fast and Lightweight Clustering of de novo Transcriptomes using Fragment Equivalence Classes<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://arxiv.org/abs/1604.03250">http://arxiv.org/abs/1604.03250</a></p>
<p><span style="font-weight: 400">Steuerman, Yael and Irit Gat-Viks.<br />
</span><i><span style="font-weight: 400">Exploiting Gene-Expression Deconvolution to Probe the Genetics of the Immune System</span></i><span style="font-weight: 400">            </span><a href="http://dx.doi.org/10.1371/journal.pcbi.1004856"><span style="font-weight: 400">http://dx.doi.org/10.1371/journal.pcbi.1004856</span></a></p>
<p><span style="font-weight: 400">Tomescu, Alexandru I., and Paul Medvedev.<br />
</span><i><span style="font-weight: 400">Safe and Complete Contig Assembly Via Omnitigs<br />
</span></i><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_11"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_11</span></a></p>
<p><span style="font-weight: 400">Tsirogiannis, Constantinos, and Brody Sandel.<br />
</span><i><span style="font-weight: 400">Fast Phylogenetic Biodiversity Computations Under a Non-Uniform Random Distribution</span></i><span style="font-weight: 400">            </span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_16"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_16</span></a></p>
<p><span style="font-weight: 400">Wang, Hao, Joel McManus and Carl Kingsford.<br />
</span><i><span style="font-weight: 400">Accurate Recovery of Ribosome Positions Reveals Slow Translation of Wobble-Pairing Codons in Yeast</span></i><span style="font-weight: 400">            </span><a href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_3"><span style="font-weight: 400">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_3</span></a></p>
<p><span style="font-weight: 400">Welch, Joshua, Ziqing Liu, Li Wang, Junjie Liu, Paul Lerou, Jeremy Purvis, Li Qian, Alexander Hartemink and Jan Prins.<br />
</span><i><span style="font-weight: 400">SLICER: Inferring Branched, Nonlinear Cellular Trajectories from Single Cell RNA-seq Data</span></i><span style="font-weight: 400">            </span> <i><span style="font-weight: 400">           </span></i><a href="http://dx.doi.org/10.1101/047845"><span style="font-weight: 400">http://dx.doi.org/10.1101/047845</span></a></p>
<p><span style="font-weight: 400">Wiedenhoeft, John, Eric Brugel and Alexander Schliep.<br />
</span><i><span style="font-weight: 400">Fast Bayesian Inference of Copy Number Variants using Hidden Markov Models with Wavelet Compression<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://biorxiv.org/content/early/2015/07/31/023705">http://biorxiv.org/content/early/2015/07/31/023705</a></p>
<p><span style="font-weight: 400">Yang, Yuqing, Ning Chen and Ting Chen.<br />
</span><i><span style="font-weight: 400">mLDM: a Statistical Model for Sparse Microbial Association Discovery<br />
</span></i><a href="http://dx.doi.org/10.1101/042630"><span style="font-weight: 400">http://dx.doi.org/10.1101/042630</span></a></p>
<p><span style="font-weight: 400">Zou, Chenchen, Yuping Zhang and Zhengqing Ouyang.<br />
</span><i><span style="font-weight: 400">Multi-track modeling for genome-scale reconstruction of 3D chromatin structure from Hi-C data</span></i><span style="font-weight: 400">            </span> <i><span style="font-weight: 400">           </span></i><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0896-1"><span style="font-weight: 400">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0896-1</span></a></p>
<p><span style="font-weight: 400">Zuo, Chandler, Kailei Chen and Sunduz Keles.<br />
</span><i><span style="font-weight: 400">A MAD-Bayes Algorithm for State-space Inference and Clustering with Application to Querying Large Collections of ChIP-Seq Data Sets<br />
</span></i><span style="font-weight: 400">           </span><a style="line-height: 1.5" href="http://link.springer.com/chapter/10.1007/978-3-319-31957-5_2">http://link.springer.com/chapter/10.1007/978-3-319-31957-5_2</a><span style="font-weight: 400">                                </span></p>
<!-- kcite active, but no citations found -->
</div> <!-- kcite-section 350 -->  </div> <!-- end .entry-content -->
  
    
</div> <!-- end #post-350 .post_class -->

<div id="comments">
  
  
  
  
</div><!-- #comments -->        
              
            
    </div> <!-- end #content -->
  </div> <!-- end .grid_10 -->
  
  <div class="grid_6 omega">
  <div id="sidebar">
  
	<div id="text-4" class="widget widget_text widget-widget_text clearfix"><div class="widget-wrap widget-inside"><h3 class="widget-title">RECOMB Schedule At-a-Glance</h3>			<div class="textwidget"><p>April 16th &amp; 17th, 2016- UCLA Campus<br />
Satellite Workshops:<br />
RECOMB-CCB<br />
RECOMB-Seq<br />
RECOMB-Genetics</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
1:30 p.m. to 6 p.m.<br />
Mike Waterman Symposium</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
2:00pm – 5:00pm<br />
Bioinformatics Flipped Course</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
6:00 p.m. to 9 p.m.<br />
RECOMB 2016 Welcome Reception</p>
<p>April 18th-21st, 2016- Loews, Santa Monica<br />
RECOMB 2016 Conference</p>
<p>Special thanks to our sponsors...</p>
<p>- International Society for Computational Biology<br />
- Polish Bioinformatics Society<br />
- 10x Genomics<br />
- USC Dornsife College of Letters, Arts and Sciences<br />
- UCLA Clinical Translational Science Institute (CTSI)<br />
- UCLA Institute for Quantitative and Computational Biosciences<br />
- Intel<br />
- PLOS Computational Biology<br />
- UCLA Department of Computer Science</p>
</div>
		</div></div>  
  </div> <!-- end #sidebar -->
</div>  <!-- end .grid_6 -->
</div> <!-- end .container_16 -->
  
  <div id="footer">
    <div class="container_16">
      <div class="grid_11">
  &copy; Copyright 2018 - <a href="index.html">Research in Computational Molecular Biology 2016</a></div>
<div class="grid_5 grid_credit">
  Retina Theme by <a href="http://wpaisle.com/" title="WPAisle">WPAisle</a> &sdot; <a href="http://wordpress.org/" title="WordPress">WordPress</a>
</div>
    </div>
  </div>

</div> <!-- end .wrapper -->
<!-- Kcite Plugin Installed: Disabled as there are no shortcodes-->
	<div style="display:none">
	</div>
<script type='text/javascript' src='http://s0.wp.com/wp-content/js/devicepx-jetpack.js?ver=201808'></script>
<script type='text/javascript' src='wp-content/themes/retina/lib/js/superfish/superfish-combine.min.js?ver=1.5.9'></script>
<script type='text/javascript' src='wp-content/themes/retina/lib/js/common.js?ver=1.0'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min.js?ver=4.4.2'></script>
<script type='text/javascript' src='http://stats.wp.com/e-201808.js' async defer></script>
<script type='text/javascript'>
	_stq = window._stq || [];
	_stq.push([ 'view', {v:'ext',j:'1:3.9.2',blog:'103766985',post:'350',tz:'0',srv:'recomb2016.bioinformatics.ucla.edu'} ]);
	_stq.push([ 'clickTrackerInit', '103766985', '350' ]);
</script>
</body>
</html>