<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head profile="http://gmpg.org/xfn/11">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>RECOMB-SEQ Schedule | Research in Computational Molecular Biology 2016</title>
<link rel="stylesheet" href="wp-content/themes/retina/style.css" type="text/css" media="all" />
<link rel="pingback" href="xmlrpc.php" />
<link rel="alternate" type="application/rss+xml" title="Research in Computational Molecular Biology 2016 &raquo; Feed" href="feed/" />
<link rel="alternate" type="application/rss+xml" title="Research in Computational Molecular Biology 2016 &raquo; Comments Feed" href="comments/feed/" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/72x72\/","ext":".png","source":{"concatemoji":"http:\/\/recomb2016.bioinformatics.ucla.edu\/wp-includes\/js\/wp-emoji-release.min.js?ver=4.4.2"}};
			!function(a,b,c){function d(a){var c,d=b.createElement("canvas"),e=d.getContext&&d.getContext("2d"),f=String.fromCharCode;return e&&e.fillText?(e.textBaseline="top",e.font="600 32px Arial","flag"===a?(e.fillText(f(55356,56806,55356,56826),0,0),d.toDataURL().length>3e3):"diversity"===a?(e.fillText(f(55356,57221),0,0),c=e.getImageData(16,16,1,1).data.toString(),e.fillText(f(55356,57221,55356,57343),0,0),c!==e.getImageData(16,16,1,1).data.toString()):("simple"===a?e.fillText(f(55357,56835),0,0):e.fillText(f(55356,57135),0,0),0!==e.getImageData(16,16,1,1).data[0])):!1}function e(a){var c=b.createElement("script");c.src=a,c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var f,g;c.supports={simple:d("simple"),flag:d("flag"),unicode8:d("unicode8"),diversity:d("diversity")},c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.simple&&c.supports.flag&&c.supports.unicode8&&c.supports.diversity||(g=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",g,!1),a.addEventListener("load",g,!1)):(a.attachEvent("onload",g),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),f=c.source||{},f.concatemoji?e(f.concatemoji):f.wpemoji&&f.twemoji&&(e(f.twemoji),e(f.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='retina-css-960-css'  href='wp-content/themes/retina/lib/css/960.css?ver=4.4.2' type='text/css' media='all' />
<link rel='stylesheet' id='retina-google-fonts-css'  href='http://fonts.googleapis.com/css?family=Open+Sans%7CComfortaa&#038;ver=4.4.2' type='text/css' media='all' />
<link rel='stylesheet' id='jetpack_css-css'  href='wp-content/plugins/jetpack/css/jetpack.css?ver=3.9.2' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery.js?ver=1.11.3'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min.js?ver=1.2.1'></script>
<link rel='https://api.w.org/' href='wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 4.4.2" />
<link rel="canonical" href="recomb-seq-schedule.html" />
<link rel='shortlink' href='http://wp.me/P71ox3-8b' />
<link rel="alternate" type="application/json+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2016.bioinformatics.ucla.edu%2Frecomb-seq-schedule%2F" />
<link rel="alternate" type="text/xml+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2016.bioinformatics.ucla.edu%2Frecomb-seq-schedule%2F&#038;format=xml" />
<style type='text/css'>img#wpstats{display:none}</style>	
		<style type="text/css">
    #headimg {
        width: 100%;
		height: 215px;
        background: url(wp-content/uploads/sites/7/2015/06/cropped-Screen-Shot-2015-06-29-at-12.56.22-PM2.png) no-repeat top center;
		-webkit-background-size: cover;
		-moz-background-size: cover;
		-o-background-size: cover;
		background-size: cover;
    }
    </style>
    	
		


<!-- Jetpack Open Graph Tags -->
<meta property="og:type" content="article" />
<meta property="og:title" content="RECOMB-SEQ Schedule" />
<meta property="og:url" content="recomb-seq-schedule.html" />
<meta property="og:description" content="RECOMB-SEQ will be held at the Faculty Center on UCLA campus: 480 Charles E Young Dr E, Los Angeles, CA 90095. &nbsp; Parking information for visitors: &nbsp; Saturday, April 16 8:30am &#8211; Brea…" />
<meta property="article:published_time" content="2016-03-10T00:47:57+00:00" />
<meta property="article:modified_time" content="2016-04-16T00:10:00+00:00" />
<meta property="og:site_name" content="Research in Computational Molecular Biology 2016" />
<meta property="og:image" content="https://s0.wp.com/i/blank.jpg" />
<meta property="og:locale" content="en_US" />
<meta name="twitter:card" content="summary" />
</head>
<body class="page page-id-507 page-template-default">
<div class="wrapper">
  
  <div id="header">  
    
    <div id="headimg"> 
  
  
</div>
          
    <div id="nav">
      <div class="container_16 clearfix">
        <div class="grid_16 alpha omega">
            <div class="menu clearfix"><ul id="menu-page-structure" class="sf-menu"><li id="menu-item-47" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-47"><a href="index.html">Home</a></li>
<li id="menu-item-49" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-49"><a href="about-recomb.html">About RECOMB</a></li>
<li id="menu-item-295" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-295"><a href="registration.html">Registration</a>
<ul class="sub-menu">
	<li id="menu-item-299" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-299"><a href="travel-support.html">Travel Support</a></li>
	<li id="menu-item-272" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-272"><a href="conference-venue.html">Conference Venue</a></li>
	<li id="menu-item-310" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-310"><a href="hotel.html">Hotel Information</a></li>
</ul>
</li>
<li id="menu-item-108" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-108"><a href="committees.html">Committees</a></li>
<li id="menu-item-81" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-81"><a href="papers-and-posters.html">Submit to RECOMB</a>
<ul class="sub-menu">
	<li id="menu-item-141" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-141"><a href="paper-submissions.html">Paper Submission</a></li>
	<li id="menu-item-138" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-138"><a href="posters.html">Poster Submission</a></li>
	<li id="menu-item-293" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-293"><a href="highlights-submission.html">Highlights Submission</a></li>
</ul>
</li>
<li id="menu-item-399" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-399"><a href="keynote-speakers.html">Program</a>
<ul class="sub-menu">
	<li id="menu-item-436" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-436"><a href="conference-schedule.html">Conference Schedule</a></li>
	<li id="menu-item-352" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-352"><a href="accepted-papers.html">Accepted Papers</a></li>
	<li id="menu-item-488" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-488"><a href="accepted-posters.html">Accepted Posters</a></li>
</ul>
</li>
<li id="menu-item-473" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-473"><a href="keynote-speakers-2.html">Keynote Speakers</a></li>
<li id="menu-item-131" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-131"><a href="satellite-meetings.html">Satellite Meetings</a>
<ul class="sub-menu">
	<li id="menu-item-132" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-132"><a href="satellite-meetings/recomb-seq.html">RECOMB-Seq</a></li>
	<li id="menu-item-529" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-529"><a href="recomb-seq-schedule.html">RECOMB-SEQ Schedule</a></li>
	<li id="menu-item-134" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-134"><a href="satellite-meetings/recomb-ccb.html">RECOMB-CCB</a></li>
	<li id="menu-item-530" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-505 current_page_item menu-item-530"><a href="recomb-ccb-schedule.html">RECOMB-CCB Schedule</a></li>
	<li id="menu-item-133" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-133"><a href="satellite-meetings/recomb-genetics.html">RECOMB-Genetics</a></li>
	<li id="menu-item-491" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-491"><a href="recomb-genetics-schedule.html">RECOMB-Genetics Schedule</a></li>
</ul>
</li>
<li id="menu-item-430" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-430"><a href="sponsors.html">Sponsors</a></li>
<li id="menu-item-302" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-302"><a href="co-founder-symposium.html">Waterman Symposium</a></li>
<li id="menu-item-376" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-376"><a href="bioinformatics-flipped-course.html">Bioinformatics Flipped Course</a></li>
</ul></div>        </div>
      </div>
    </div>
  
  </div>
<div class="container_16 clearfix">
  
  <div class="grid_10 alpha">
    <div id="content">	  
	  
	        
                
          <div id="post-507" class="post-507 page type-page status-publish hentry">
  
    <h1 class="entry-title entry-title-single entry-title-page">RECOMB-SEQ Schedule</h1>
  
    
  <div class="entry-content clearfix">
  	<div class="kcite-section" kcite-section-id="507">
<p>RECOMB-SEQ will be held at the Faculty Center on UCLA campus: 480 Charles E Young Dr E, Los Angeles, CA 90095.</p>
<div class="googlemaps"><iframe width="400" height="400" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d13219.891748367743!2d-118.439237!3d34.070208!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0:0x9251f353871bb1af!2sUCLA Faculty Center!5e0!3m2!1sen!2sus!4v1459473026108"></iframe></div>
<p>&nbsp;</p>
<p>Parking information for visitors:<br />
<a href="https://main.transportation.ucla.edu/campus-parking/visitors" target="_blank">https://main.transportation.ucla.edu/campus-parking/visitors</a></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">Saturday, April 16 </span></p>
<p><strong><span class="aBn"><span class="aQJ">8:30am</span></span> &#8211; Breakfast</strong><br />
UCLA Faculty Center [<a href="http://maps.ucla.edu/campus/?locid=342" target="_blank">view on UCLA map</a>]<br />
480 Charles E Young Dr E, Los Angeles, CA 90095</p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">8:50 &#8211; 9:00 a.m. &#8211; Opening remarks by organizing committee</span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">9:00 &#8211; 9:50 a.m. &#8211; KEYNOTE: Jared Simpson</span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">9:55 &#8211; 10:35 a.m. &#8211; Contributed Talks:</span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400"> Leena Salmela, Riku Walve, Eric Rivals and Esko Ukkonen. </span><i><span style="font-weight: 400">Accurate self correction of errors in long reads using de Bruijn graphs </span></i></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">Joaquín Ezpeleta, Flavia J. Krsticevic, Pilar Bulacio and Elizabeth Tapia.</span><i><span style="font-weight: 400"> Designing robust watermark barcodes for multiplex long-read sequencing</span></i></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">10:35 &#8211; 10:50 a.m. &#8211; Coffee break </span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">10:50 &#8211; 11:15 a.m. &#8211; Contributed Talks:</span></p>
<p><span style="font-weight: 400">Lorenzo Calviello, Neelanjan Mukherjee, Emanuel Wyler, Henrik Zauber, Antje Hirsekorn, Matthias Selbach, Markus Landthaler, Benedikt Obermayer and Uwe Ohler. Detecting actively translated open reading frames in ribosome profiling data.</span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">11:15 &#8211; 12:15 p.m. &#8211; Poster Flash Talks</span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">12:15 &#8211; 2:00 p.m. &#8211; Lunch &amp; posters </span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">2:00 &#8211; 3:00 p.m. &#8211; Contributed talks:</span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">Yang Lu, Ting Chen, Jed Fuhrman and Fengzhu Sun. </span><i><span style="font-weight: 400">COCACOLA: binning metagenomic contigs using sequence COmposition, read CoverAge, CO-alignment, and paired-end read LinkAge</span></i></p>
<p><i><span style="font-weight: 400"><br />
</span></i><span style="font-weight: 400">Roye Rozov, Aya Brown Kav, David Bogumil, Itzhak Mizrahi, Eran Halperin and Ron Shamir. </span><i><span style="font-weight: 400">Recycler: an algorithm for detecting plasmids from de novo assembly graphs</span></i></p>
<p><i><span style="font-weight: 400"><br />
</span></i><span style="font-weight: 400">Dmitry Antipov, Nolan Hartwick, Max Shen, Michael Rayko, Alla Lapidus and Pavel Pevzner. </span><i><span style="font-weight: 400">plasmidSPAdes: Assembling Plasmids from Whole Genome Sequencing Data</span></i><i></i></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">3:00 &#8211; 3:15 p.m. &#8211; Coffee break </span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">3:15 &#8211; 4:05 p.m. &#8211; KEYNOTE: Christina Curtis, PhD<br />
Stanford University School of Medicine, Departments of Medicine and Genetics<br />
</span></p>
<p><em>Quantifying the evolutionary dynamics of human tumor growth and progression</em></p>
<p>Genetic diversification and clonal selection underlie tumor progression and resistance,<br />
but their dynamics are poorly characterized. Although direct observations of human<br />
tumor growth are impractical, tumors faithfully record their ancestries in the form of<br />
somatic alterations acquired during cell division such that the resultant patterns of intratumor<br />
heterogeneity can be exploited through multi-region sequencing to infer the<br />
evolutionary history of the tumor. Recently, we described a novel ‘Big Bang’ model of<br />
primary human colorectal tumor growth, whereby after transformation, the neoplasm<br />
grows predominantly as a single terminal expansion producing numerous intermixed<br />
sub-clones that are not subject to stringent selection and where both public and most<br />
detectable private alterations arise early during growth. Hence, in this model, the timing<br />
of a mutation is the fundamental determinant of its frequency in the final tumor. Through<br />
spatial computational modeling and statistical inference we verify that the majority of<br />
detectable intra-tumor heterogeneity occurs during the earliest stages of tumor growth.<br />
This new model is compatible with effectively neutral evolution and provides a<br />
framework for quantifying the dynamics of tumor growth with fundamental implications<br />
for earlier detection, therapeutic resistance and metastasis. I will describe extensions of<br />
our computational models to delineate mechanisms of disease progression and discuss<br />
the importance of accounting for tumor growth dynamics when interpreting cancer<br />
genomic data.</p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">4:05 &#8211; 4:25 &#8211; Coffee break </span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">4:25 &#8211; 5:30 &#8211; Joint Contributed Talks: </span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">Nisha Rajagopal, Sharanya Srinivasan, Kameron Kooshesh, Yuchun Guo, Matthew Edwards, Budhaditya Banerjee, Tahin Syed, Bart Emons, David Gifford and Richard Sherwood. High-throughput mapping of regulatory DNA</span></p>
<p><span style="font-weight: 400"><br />
</span><span style="font-weight: 400">Avi Srivastava, Hirak Sarkar, Laraib Malik and Robert Patro. </span><i><span style="font-weight: 400">Fast, Lightweight Clustering of de novo Transcriptomes using Fragment Equivalence Classes</span></i></p>
<p><span style="font-weight: 400"><br />
</span><span style="font-weight: 400">Mikhail Kolmogorov, Eamonn Kennedy, Zhuxin Dong, Gregory Timp and Pavel Pezvner. </span><i><span style="font-weight: 400">Single-Molecule Protein Identification by Sub-Nanopore Sensors</span></i><i></i></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">Sunday April 17th</span></p>
<p><strong><span class="aBn"><span class="aQJ">8:30am</span></span> &#8211; Breakfast</strong><br />
UCLA Faculty Center [<a href="http://maps.ucla.edu/campus/?locid=342" target="_blank">view on UCLA map</a>]<br />
480 Charles E Young Dr E, Los Angeles, CA 90095</p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">9:00 &#8211; 9:50 a.m. &#8211; Keynote Speaker: Christina Boucher. </span><em>Systematic Surveillance of the Resistome in Food Production</em></p>
<p>The World Health Organization describes antimicrobial resistance (AMR) as “an increasingly serious threat to global public health.” This threat has prompted the President of the United States to issue an executive order initiating the National Action Plan for Combating Antibiotic-Resistant Bacteria. The fifth goal of this action plan is to: “improve international collaboration and capacities for antibiotic-resistance prevention, surveillance, control, and antibiotic research and development.” This plan also recognizes that humans, animals, and the environment can be sources of AMR, and calls for a “One-Health approach to disease surveillance.” However, despite the growing concern with antibiotic use in agriculture, there are very few computational methods to measure, quantify, and track the AMR genes within a food production system. In this talk, we present a computational framework that will detect, quantify, and track AMR agents contained in, and around, a food production facility.</p>
<p>Our approach focuses on the systematic surveillance of the microbiome—or more specifically the resistome—within a food production system through the analysis of high throughput sequence data. The computational methods that are part of this approach rely on the use of succinct data structures that dramatically reduce the amount of memory required to store and use the de Bruijn graph. Thus, allowing it to be applied in much larger and more ambitious sequence projects than was previously possible, such as the widespread surveillance of the resistome. In addition to surveillance, we present tools to compare the resistome of different agricultural production systems to identify possible sources of AMR within these systems. Although our methods are developed for the study of the resistome in food production, our methods are generalizable to other agricultural or human settings.</p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">9:50 &#8211; 10:50 a.m. &#8211; Contributed Talks:</span></p>
<p><span style="font-weight: 400"><br />
</span><span style="font-weight: 400">Alexander Shlemov, Sergey Bankevich, Andrey Bzikadze and Yana Safonova.</span><i><span style="font-weight: 400"> IgReC 2.0: New algorithmic challenges of adaptive immune repertoire construction</span></i></p>
<p><span style="font-weight: 400"><br />
</span><span style="font-weight: 400">Nitish Gupta, Komal Sanjeev, Tim Wall, Carl Kingsford and Rob Patro. </span><i><span style="font-weight: 400">Efficient Index Maintenance Under Dynamic Genome Modification</span></i></p>
<p><i><span style="font-weight: 400"><br />
</span></i><span style="font-weight: 400">Sunyoung Kwon, Gyuwan Kim, Byunghan Lee, Sungroh Yoon and Young-Han Kim. </span><i><span style="font-weight: 400">NASCUP: Nucleic Acid Sequence Classification by Universal Probability</span></i></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">10:50 &#8211; 11:05 a.m. &#8211; Coffee break </span></p>
<p>&nbsp;</p>
<p><span style="font-weight: 400">11:05 &#8211; 12:05 p.m. &#8211; Panel of Journal Editors </span><br />
<span style="font-weight: 400">12:05 p.m. &#8211; End of Program </span><i><span style="font-weight: 400"><br />
</span></i></p>
<!-- kcite active, but no citations found -->
</div> <!-- kcite-section 507 -->  </div> <!-- end .entry-content -->
  
    
</div> <!-- end #post-507 .post_class -->

<div id="comments">
  
  
  
  
</div><!-- #comments -->        
              
            
    </div> <!-- end #content -->
  </div> <!-- end .grid_10 -->
  
  <div class="grid_6 omega">
  <div id="sidebar">
  
	<div id="text-4" class="widget widget_text widget-widget_text clearfix"><div class="widget-wrap widget-inside"><h3 class="widget-title">RECOMB Schedule At-a-Glance</h3>			<div class="textwidget"><p>April 16th &amp; 17th, 2016- UCLA Campus<br />
Satellite Workshops:<br />
RECOMB-CCB<br />
RECOMB-Seq<br />
RECOMB-Genetics</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
1:30 p.m. to 6 p.m.<br />
Mike Waterman Symposium</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
2:00pm – 5:00pm<br />
Bioinformatics Flipped Course</p>
<p>April 17th, 2016- Loews, Santa Monica<br />
6:00 p.m. to 9 p.m.<br />
RECOMB 2016 Welcome Reception</p>
<p>April 18th-21st, 2016- Loews, Santa Monica<br />
RECOMB 2016 Conference</p>
<p>Special thanks to our sponsors...</p>
<p>- International Society for Computational Biology<br />
- Polish Bioinformatics Society<br />
- 10x Genomics<br />
- USC Dornsife College of Letters, Arts and Sciences<br />
- UCLA Clinical Translational Science Institute (CTSI)<br />
- UCLA Institute for Quantitative and Computational Biosciences<br />
- Intel<br />
- PLOS Computational Biology<br />
- UCLA Department of Computer Science</p>
</div>
		</div></div>  
  </div> <!-- end #sidebar -->
</div>  <!-- end .grid_6 -->
</div> <!-- end .container_16 -->
  
  <div id="footer">
    <div class="container_16">
      <div class="grid_11">
  &copy; Copyright 2018 - <a href="index.html">Research in Computational Molecular Biology 2016</a></div>
<div class="grid_5 grid_credit">
  Retina Theme by <a href="http://wpaisle.com/" title="WPAisle">WPAisle</a> &sdot; <a href="http://wordpress.org/" title="WordPress">WordPress</a>
</div>
    </div>
  </div>

</div> <!-- end .wrapper -->
<!-- Kcite Plugin Installed: Disabled as there are no shortcodes-->
	<div style="display:none">
	</div>
<script type='text/javascript' src='http://s0.wp.com/wp-content/js/devicepx-jetpack.js?ver=201808'></script>
<script type='text/javascript' src='wp-content/themes/retina/lib/js/superfish/superfish-combine.min.js?ver=1.5.9'></script>
<script type='text/javascript' src='wp-content/themes/retina/lib/js/common.js?ver=1.0'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min.js?ver=4.4.2'></script>
<script type='text/javascript' src='http://stats.wp.com/e-201808.js' async defer></script>
<script type='text/javascript'>
	_stq = window._stq || [];
	_stq.push([ 'view', {v:'ext',j:'1:3.9.2',blog:'103766985',post:'507',tz:'0',srv:'recomb2016.bioinformatics.ucla.edu'} ]);
	_stq.push([ 'clickTrackerInit', '103766985', '507' ]);
</script>
</body>
</html>
